rabbit_mq/provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  profile = var.profile
+  region  = var.region
```

Comment:
Need to remove region here and in variables file if  it is not specified elsewhere. We are to use AWS_DEFAULT_REGION env variable now to specify region. This should be added in the docs as well

Reply:
removed